src/org/omegat/util/TMXReader2.java

```diff
@@ -160,7 +160,11 @@ public void readTMX(File file, final Language sourceLanguage, final Language tar
                     if ("tu".equals(eStart.getName().getLocalPart())) {
                         parseTu(eStart);
                         ParsedTuv origTuv = getTuvByLang(sourceLanguage);
+                        // This ParsedTuv for the target translation is not used, but is there to avoid
+                        // any NPE during the callback.onEntry below.
                         ParsedTuv targetTuv = getTuvByLang(targetLanguage);
+                        // The filtering of the target Tuvs is done in
+                        // ExternalTMFactory.TMXLoader.loadImpl => LoadCallback.onEntry,
```

Comment:
`TMXReader2` is not only used by `ExternalTMFactory`, so I don't think either of these comments are appropriate here.

Reply:
Comment removed.